client/me/purchases/cancel-purchase/style.scss

```diff
@@ -25,7 +25,7 @@
 	}
 
 	p {
-		color: var(--color-neutral-light);
+		color: var(--color-text-subtle);
```

Comment:
Is this the one you meant to change, or is it the one further up (under `.cancel-purchase__card`)?

It's possible I'm missing it, but I can't seem to find any difference as a result of this change.